packages/desktop-client/src/components/mobile/MobileNavTabs.tsx

```diff
@@ -43,6 +43,43 @@ export function MobileNavTabs() {
     padding: 10,
   };
 
+  const [{ y }, api] = useSpring(() => ({ y: OPEN_DEFAULT_Y }));
```

Comment:
Technically there are no changes here. I just re-arranged the constant definitions so that `openDefault` could be accessed by the `NavLink` definitions. It will now close the bottom nav bar whenever a item in the nav is clicked (thus navigating to a different page).

This should make the VRT tests more stable. 🤞